cincinnati/Cargo.toml

```diff
@@ -5,45 +5,45 @@ authors = ["Alex Crawford <crawford@redhat.com>"]
 edition = "2018"
 
 [dependencies]
-actix-web = "^3.3.2"
+actix-web = "^4.0.0-beta.3"
```

Comment:
are we sure we want to upgrade to a beta version?

Reply:
Yeah, beta versions are the only ones using tokio 1.0.

It appears these have been stabilized for quite time, so I'm not too worried about it